ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -560,16 +560,25 @@ else
 	fi
 
 	#
-	# Start mavlink streams.
+	# Waypoint storage.
+	# REBOOTWORK this needs to start in parallel.
 	#
-	sh /etc/init.d/rc.mavlink
+	dataman start $DATAMAN_OPT
```

Comment:
dataman needs to be started early on, because other modules such as mavlink, navigator and commander depend on it.

Reply:
I moved it up high in rcS just below uorb, tone_alarm, tune_control startups to start to untangle the locations of startups in rcS.  My intention is to try make debugging easier in the future by grouping. If I've missed another detail preventing moving them up this high in the script just let me know!  Here is a compare of the current:
![image](https://user-images.githubusercontent.com/2497951/42899156-61af0be2-8a82-11e8-9f6b-68f75f73e946.png)